package.json

```diff
@@ -19,6 +19,19 @@
   ],
   "main": "src/lib/main.js",
   "types": "src/lib/main.d.ts",
+  "exports": {
+    ".": {
+      "types": "./src/lib/main.d.ts",
+      "default": "./src/lib/main.js"
+    },
+    "./config": {
+      "types": "./src/lib/config.d.ts",
+      "default": "./src/lib/config.js"
+    },
+    "./package.json": {
+      "default": "./src/lib/main.js"
```

Comment:
I'm not sure whether there also needs to be a `types` sibling import condition here in order to appease TypeScript imports of `@dotenvx/dotenvx/package.json`, or whether TypeScript is already infers the types from the contents of `package.json`.

Reply:
Resolving this one as well... TypeScript requires [`--resolveJsonModule`](https://www.typescriptlang.org/tsconfig/#resolveJsonModule) in order to be able to import JSON files, and the type is generated from JSON:

> This includes generating a type for the `import` based on the static JSON shape.